Assets/MixedRealityToolkit/MixedReality.Toolkit.nuspec

```diff
@@ -1,16 +1,16 @@
 <?xml version="1.0"?>
 <package >
   <metadata>
-    <id>Microsoft.MixedReality.Toolkit</id>
+    <id>Microsoft.MixedReality.Toolkit.Unity.Core</id>
```

Comment:
Modifying this breaks the MixedReality.Toolkit.SDK dependency, which is set for Microsoft.MixedReality.Toolkit.